src/commands/fun/twitch.js

```diff
@@ -0,0 +1,62 @@
+const SA = require('superagent')
+const Command = require('../../classes/Command')
+
+module.exports = new Command(async (msg, suffix) => {
+	if (!suffix) return msg.channel.createMessage('No channel specified!')
+	const user = await getUser(suffix);
+	if(!user) return msg.channel.createMessage(`**${suffix}** isn't a valid channel.`)
+	SA.get(`https://api.twitch.tv/kraken/streams/${user}`)
+		.set({ 'Accept': 'application/vnd.twitchtv.v5+json', 'Client-ID': process.env.TWITCH_ID })
+		.end((err, response) => {
+			if (err) {
+				global.logger.error(err)
+				return msg.channel.createMessage('We ran into an error, sorry about that!')
+			}
+			response.body.stream ? msg.channel.createMessage(getEmbed(response.body)) :
+				msg.channel.createMessage(`**${suffix}** is not currently streaming.`)
```

Comment:
Although our style guide does not appear to be strictly enforcing this (why?), inconsistent ternary line breaks severely hamper readability

```suggestion
			response.body.stream
				? msg.channel.createMessage(getEmbed(response.body))
				: msg.channel.createMessage(`**${suffix}** is not currently streaming.`)
```

Reply:
[We do enforce ternary style](https://eslint.org/docs/rules/operator-linebreak)
![afbeelding](https://user-images.githubusercontent.com/9768134/80864759-ec860800-8c84-11ea-84ff-5d1ce059ce24.png)

Reply:
🤔 then why did ESLint not complain about it for me